saleor/plugins/avatax/excise/tests/test_avatax_excise.py

```diff
@@ -0,0 +1,201 @@
+import pytest
+from prices import Money, TaxedMoney
+
+from .....account.models import Address
+from .....checkout import CheckoutLineInfo
+from .....checkout.utils import add_variant_to_checkout, fetch_checkout_lines
+from .....core.prices import quantize_price
+from .....product.models import ProductVariant
+from .....warehouse.models import Warehouse
+from ....manager import get_plugins_manager
+from ....models import PluginConfiguration
+from .. import get_metadata_key
+from ..plugin import AvataxExcisePlugin
+
+
+@pytest.fixture
+def plugin_configuration(db):
```

Comment:
Have you checked the fixture build for Adyen? `saleor.payment.gateways.adyen.tests.conftest.adyen_plugin`
I would say, that the adyen fixture is easier to use. You get the plugin directly from the fixture, so no need to call manager and define path to plugin in each call.

Reply:
IMO let's take the time to update the avatax plugin or let's use the older ways. We're basing this plugin on Avatax. Refactoring parts of it but not in Avatax IMO makes the code more confusing to read because now I have to understand the old and new methods and I can't easily tell if the ATE plugin was modified because it has to be or because it was some refactor.

What's the right workflow here? Can you make the changes to Avatax and we can rebase? Should we?  Of course I prefer you to :)

Reply:
I don't think that we have enough resources to update the Avatax for now. It is planned but without eta